### DISCUSSION TITLE
my lazyvim logo is aint visible and there are bunch of things in the GUI is a bit glitching

### DISCUSSION THREAD

**Discussion by rshdhere** [LazyVim/LazyVim#4030] [Q&A] [answered]
my lazyvim logo is aint visible and there are bunch of things in the GUI is a bit glitching
![Screenshot from 2024-07-14 06-51-35](https://github.com/user-attachments/assets/631b9c5d-1178-46b7-8953-c5905b9c32a2)
![Screenshot from 2024-07-14 15-15-51](https://github.com/user-attachments/assets/382ccbb1-c78c-4eaf-8244-49d1befa6800)
it shows something like this
expecting a guidiance

**Comment by folke** [comment]
Use a proper terminal. Check the docs

**Reply by rshdhere** [reply]
Im on ubuntu 24.10 actually, can you attach the exact doc link for my glitch 
it would be good for me

**Reply by dpetka2001** [reply, marked as answer]
Try to use either `kitty` or `wezterm` terminal with recently updated Nerd fonts.

**Reply by rshdhere** [reply]
thank you smm
